core/src/main/java/hudson/scm/ChangeLogParser.java

```diff
@@ -52,6 +63,16 @@
         }
     }
 
+    /**
+     * Method for compatibility with AbstractBuild.
```

Comment:
This is a good point. Maybe it makes sense to name the new method as `parseRun()` to prevent ambiguity. Then all internal calls may be safely moved to the new method